cli/src/fmt/simple/mod.rs

```diff
@@ -6,16 +6,73 @@ macro_rules! append_to_buffer {
     }
 }
 
-pub struct Simple;
+macro_rules! nest_in_buffer {
```

Comment:
I wonder if we can remove the `_in_buffer` and `_to_buffer` so that it's more like a dsl and just has 3 verbs: `append!`, `indent!`, and `nest!`.

I believe it's properly scoped to only the simple module so it shouldn't harm any other modules.

Reply:
I like this a lot better! I was finding the `_in_buffer` naming awkward.